cmd/infra/powervs/create.go

```diff
@@ -326,7 +325,8 @@ func (infra *Infra) setupBaseDomain(options *CreateInfraOptions) (err error) {
 	}
 
 	if rcv2 == nil {
-		return fmt.Errorf("unable to get resource controller")
+		err = fmt.Errorf("unable to get resource controller")
```

Comment:
why this change? the previous is consistent with the style across the code base.